### PR TITLE
refactor: rebrand to web3auth

### PR DESCRIPTION
**Pull request by michaellee8**

### CODE REVIEW

**Review by tanujgupta — APPROVED** [review]

File: Web3Auth.podspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  spec.name          = "Web3Auth"
+  spec.version       = "3.0.0"
+  spec.platform      = :ios, "12.0"
+  spec.summary       = "Torus Web3Auth SDK for iOS applications"
+  spec.homepage      = "https://github.com/torusresearch/web3auth-swift-sdk"
+  spec.license       = { :type => 'MIT', :file => 'LICENSE.md' }
+  spec.swift_version = "5.0"
+  spec.author        = { "Torus Labs" => "hello@tor.us" }
+  spec.module_name   = "Web3Auth"
+  spec.source        = { :git => "https://github.com/torusresearch/web3auth-swift-sdk.git", :tag => spec.version }
```
Comment (chaitanyapotti) [inline Web3Auth.podspec:11]: this would change to https://github.com/web3auth/web3auth-swift-sdk
Pls replace this everywhere

**Review by chaitanyapotti — CHANGES_REQUESTED** [review]
pls change the url

**Comment by chaitanyapotti** [comment]
Also, release and update the version of fetch-node-details package

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**